src/main/java/teammates/ui/template/FeedbackSubmissionEditQuestion.java

```diff
@@ -35,6 +38,9 @@ public FeedbackSubmissionEditQuestion(FeedbackQuestionAttributes questionAttribu
         isRecipientNameHidden = questionAttributes.isRecipientNameHidden();
         isGiverTeam = questionAttributes.giverType.equals(FeedbackParticipantType.TEAMS);
         isRecipientTeam = questionAttributes.recipientType.isTeam();
+        isRecipientOwnTeamMembers = questionAttributes.recipientType.isOwnTeamMembers();
+        isRecipientOtherTeams = questionAttributes.recipientType.isOtherTeams();
```

Comment:
There are 2 things to determine: 
1. Whether or not to show the message: this is done by the check you included in the tag file, and is independent of recipient type
2. The message to display to the user, this is dependent on the recipient type. I don't think we should try to do this case-by-base for each possible participant type. My initial comment to avoid this was to suggest we use more coarse-grained groups (`isTeam`, etc) of the participant types to determine what to show. Since the messages look very identical other than the participant type, instead of writing a message for the different types specified, maybe first we can try using a more generic message and the `displayNameVisibility` field of the `FeedbackParticipantType`?